### DISCUSSION TITLE
I can not find the command how to create a datatable

### DISCUSSION THREAD

**Discussion by ArtMin96** [rappasoft/laravel-livewire-tables#368] [Q&A] [answered]
I can not find the command how to create a datatable

**Comment by rappasoft** [comment, marked as answer]
There are no commands. Take a look at the examples:

https://github.com/rappasoft/laravel-livewire-tables/wiki/Basic-Example-Table
https://github.com/rappasoft/laravel-livewire-tables/wiki/Advanced-Example-Table

And the rest of the wiki for configuration.

**Reply by ArtMin96** [reply]
Thanks
